Makefile

```
@@ -70,12 +70,20 @@ pre-commit: $(NODE_MODULES) test-min lint
 clean:  ## Cleans up everything
 	rm -rf node_modules .*-updated .*-bin out
 
-.PHONY: run
-run: prereqs  ## Runs the site like it runs in production
+.PHONY: prebuild
+prebuild: prereqs
 	$(NPM) run webpack
 	$(NPM) run ts-compile
+
+.PHONY: run-only
+run-only: node-installed  ## Runs the site like it runs in production without building it
 	env NODE_ENV=production $(NODE) $(NODE_ARGS) -r esm ./out/dist/app.js --webpackContent ./out/webpack/static $(EXTRA_ARGS)
 
+.PHONY: run
+run:  ## Runs the site like it runs in production
```

```suggestion
run: prebuild   ## Runs the site like it runs in production
```

and then we only re-invoke make for `run-only` here?

---

Don't think it makes much difference so can do if you'd rather, just felt it was a tiny bit cleaner to be consistent between the two stages

---

Fair enough! Will merge as-is! thanks!